<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>image_mover_mult</class>
 <widget class="QWidget" name="image_mover_mult">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <column>
      <property name="text">
       <string>Image name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Move to folder(int)</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Image folder</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditImageFolder"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSelectImageFolder">
       <property name="text">
        <string>Select folder</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButtonSelectPage">
       <property name="text">
        <string>Select page</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="checkBoxLoadImageRecursive">
       <property name="text">
        <string>Load image recursive</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButtonMoveImages">
       <property name="text">
        <string>Move images</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButtonLoadImages">
       <property name="text">
        <string>Load images</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButtonSetFolderMoveTo">
       <property name="text">
        <string>Set folder move to</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QSpinBox" name="spinBoxSelectPage"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
